### DISCUSSION TITLE
I renamed github action, we have a workflow with new name, but the old name keep appearing as a separate workflow

### DISCUSSION THREAD

**Discussion by nam-97** [community/community#102079] [Actions] [answered]
### Select Topic Area

Bug

### Body

- In the past, we used to have a workflow name `pr_to_master` to run every time a PR is opened
- Recently, we renamed `pr_to_master` to `ci`, and things work well with the new name in the workflow
- Yesterday, when migrating the repo to a new organization, we have the following issue
   - workflow `ci` still runs as usual when opening PR
   - however, we have a workflow name `pr_to_master`, without any detail, and always has status `running`
![image](https://github.com/community/community/assets/127033868/b9fefa3e-0748-4734-a5eb-a57fc946b69f)

We just want to know are there any ways to remove the workflow name `pr_to_master`?

**Comment by msuliq** [comment, marked as answer]
Hello,
Here is a post for similar issue, even though it is not answered, it might provide some value in identifying the root cause: https://github.com/orgs/community/discussions/42874

Personally, I can suggest following steps:
Check Workflow Files: ensure that in the new organization, you don't have any outdated or conflicting workflow files that might be triggering the `pr_to_master` workflow. You can check both the old and new organization repositories for any lingering workflow files.

Search for References: perform a search across your entire codebase in the new organization for references to the old workflow name. There might be instances in code or configuration files that are still pointing to the old workflow name.

Check Branch Protection Rules: (this mentioned in the link too) if your organization has branch protection rules set up, ensure that the protection rules for the default branch (like master or main) are not triggering the `pr_to_master workflow` unintentionally.

Review GitHub Actions Settings: In the GitHub repository settings, navigate to the "Actions" tab and review the workflow settings. Check for any references to the old workflow name and update them to the new name (`ci`).

Manual Trigger: Manually trigger the problematic workflow (`pr_to_master`) and observe the logs. This might provide additional information on the root cause.

GitHub Actions API: you can use GitHub Actions API to inspect the workflows and runs. You can use the API to get details about workflow runs, their status, and any associated errors.

Workflow Visualizer: GitHub also provides a visualizer for GitHub Actions workflows. You can use it to visually inspect the workflow and identify any issues. You can find it under the "Actions" tab in your repository.

If the issue persists, and you are unable to identify the root cause, consider reaching out to GitHub Support. You can submit a request through the GitHub Support website: https://support.github.com/?tags=dotcom-direct

If you find this answer useful I would very much appreciate if you could mark it as accepted since it will help me get the galaxy brain badge. Thank you

**Reply by nam-97** [reply]
Hi @msuliq, thank you for the detailed answer, turns out that we forgot to check Branch Protection Rules

**Reply by msuliq** [reply]
Glad it helped :)

**Reply by keith-minnig** [reply]
Thanks. That helped point me in the direction of deleting old workflow logs.

cli command to remove them:
```for i in `gh run list --workflow "NAME OF OLD WORKFLOW" --jq '.[].databaseId' --json databaseId` ; do echo $i ; gh run delete $i ; done```
